src/components/interaction-designer/blocks/BlockToolbar.vue

```diff
@@ -1,9 +1,5 @@
 <template>
-  <div :class="{
```

Comment:
The cool thing about moving the block-toolbar inside the `draggable-handle` is to:
- inherit the Block.vue's style instead of creating new colour style here
- not worry about the toolbar position, margin, etc

Meaning, we can remove all the scoped css for this component. 🤟 🥳